app/controllers/api/v1/npq_participants_controller.rb

```diff
@@ -6,24 +6,28 @@ class NPQParticipantsController < Api::ApiController
       include ApiTokenAuthenticatable
       include ApiPagination
       include ApiFilter
+      include Api::ParticipantActions
 
       def index
-        npq_participant_hash = NPQParticipantSerializer.new(paginate(npq_participants)).serializable_hash
-        render json: npq_participant_hash.to_json
+        render json: NPQParticipantSerializer.new(paginate(npq_profiles)).serializable_hash.to_json
       end
 
     private
 
+      def serialized_response(profile)
+        NPQParticipantSerializer
+          .new(profile.validation_data)
+          .serializable_hash.to_json
+      end
```

Comment:
In general a 'participant' is a domain term we use to describe a user who has a particular type of profile. So ECF participant is a user who has an ECF profile (either a mentor or an ECT). And an NPQ participant is a user who has an NPQ profile (one or more).

So it looks weird to me to see `NPQParticipantSerializer` take what's essentially an npq application. Why is it not taking a user object?